design/ceph/external-cluster-monitoring.md

```diff
@@ -0,0 +1,63 @@
+---
+title: external-cluster-monitoring
```

Comment:
We haven't used these title tags on design docs before, just on the docs since they are generated based on the special tags. Why add them here?

Reply:
It's just a convention I've proposed here: https://github.com/rook/rook/pull/5616 but I can remove it if it's not useful. Let me know.